integration/firestore/bundle_test_data/generate_test_bundle.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * The main function for this script.
+ * @param {array} args The command line arguments.
+ */
+import * as firestore from '@firebase/firestore-types';
+import { TestBundleBuilder } from '@firebase/firestore/test/unit/util/bundle_data';
+import { DatabaseId } from '@firebase/firestore/src/core/database_info';
+import { key } from '@firebase/firestore/test/util/helpers';
+import { collectionReference } from '@firebase/firestore/test/util/api_helpers';
+import {
+  EMULATOR_PROJECT_ID,
+  ALT_PROJECT_ID
+} from '@firebase/firestore/test/integration/util/emulator_settings';
```

Comment:
Your import make a pretty strong case that this file should live under packages/firestore (probably `packages/firestore/scripts`. Once you do that, you should be able to use `ts-node` as well and remove the Rollup config.

Reply:
Done.